extensions/amp-story/0.1/amp-story-desktop.css

```diff
@@ -446,7 +446,12 @@ amp-story-page.i-amphtml-story-page-shown {
 [desktop] .i-amphtml-story-system-layer .i-amphtml-story-ui-left {
   display: block!important;
   float: left!important;
-  margin: 16px 40px!important;
+  height: 40px;
+  position: absolute;
+  top: 0;
+  bottom: 0;
+  margin: auto;
+  left: calc(50% - 17vw - 72px);
```

Comment:
Please add a comment explaining this value.

Reply:
Actually, could we maybe restructure this so that positioning isn't absolute? Using flexbox perhaps?

Reply:
not really sure, because this depends on the mid card size... which itself is not implemented by flexbox, so hard to predict

Reply:
oh, my bad... got confused between PRs... probably yeah flexbox might work here

Reply:
@alanorozco Done!